manticore/core/smtlib/solver.py

```diff
@@ -496,18 +497,24 @@ def optimize(self, constraints: ConstraintSet, x: BitVec, goal: str, M=10000):
                         maybe_sat = self._recv()
                         if maybe_sat == "sat":
                             m = RE_MIN_MAX_OBJECTIVE_EXPR_VALUE.match(_status)
-                            expr, value = m.group("expr"), m.group("value")
-                            assert expr == aux.name
-                            return int(value)
+                            if m:
+                                expr, value = m.group("expr"), m.group("value")
+                                assert expr == aux.name
+                                return int(value)
+                            else:
```

Comment:
Looks fine, but I'm guessing this is unrelated to the rest of the PR?

Reply:
Actually, it's something that mypy caught, since the return type of `regex.match()` could be `None` (type `Optional`), so in order to pass mypy, I added this change and the similar one a couple lines down.

I can make a separate PR for these changes, if you'd like.